flask_restalchemy/serialization/tests/test_serializer.py

```diff
@@ -11,13 +13,12 @@
     "1994-07-17T20:53:00.000"
 ])
 def test_datetime(serialized_date):
-    serializer = DateTimeSerializer()
+    serializer = DateTimeSerializer(Column(UtcDateTime))
     date_obj = datetime(1994, 7, 17, 20, 53)
     assert date_obj == serializer.load(serialized_date)
 
-
 def test_datetime_with_sec_tz():
-    serializer = DateTimeSerializer()
+    serializer = DateTimeSerializer(Column(UtcDateTime))
```

Comment:
I don't think we should add a new dependency just for testing purposes. Use default SQLAlchemy `DateTime` instead.